src/app/connect/embedded-wallet/custom-auth/configuration/page.mdx

```diff
@@ -128,7 +101,40 @@ You can set your encryptionKey in either a secret env variable or ask the user t
 
 ** Do not hardcode the encryptionKey in your code. **
 
-#### Generic auth
+## Generic auth
+
+Generic auth is a lower level option that can be used when you have your own auth server that you use to authenticate users.
+
+Generic auth is good for supporting other auth method that we don't support today, like discord, twitter, github etc., or something completely custom.
+
+High level flow and requirement for generic auth via endpoint:
+
+- When a user logs in, you are able to generate a public identifier that allows you to identify the user.
+- You can pass this identifier to the embedded wallet to generate a wallet for the user.
+- When verifying the user, we will hit an endpoint that you provide to verify the user's identity.
+- We will then generate a wallet for the user if the provided payload is valid.
+
+<DocImage src={EWCustomAuth} />
+
+To use generic auth, enable the second option in the configuration tab of the embedded wallet dashboard
+
+<DocImage src={EWCustomAuthdb2} />
+
+You will be asked to enter an endpoint that we can hit to verify the user's identity. This endpoint should accept a POST request with a JSON body containing the following fields:
+
+- `payload`: This will correspond to the public identifier that was generated for your user.
```

Comment:
should prob mention this is the payload that is passed in from the client side SDK